.github/workflows/tests.yml

```diff
@@ -1,51 +1,52 @@
-name: tests
+name: "tests"
 
 on:
   push:
   pull_request:
```

Comment:
We should also change something here. For a PR whole GA runs twice. One for push one for PR. IS there a way to avoid that @szepeviktor ?

Reply:
You know I'm a Travis guy. I think GHA is nowadays' hype train.

How about listing "all" our branches in `push:` ? Meaning `master`.

Reply:
Worth a try I guess :man_shrugging: